### DISCUSSION TITLE
How to implement spinner functionality

### DISCUSSION THREAD

**Discussion by bigfatwhale** [elimintz/justpy#356] [Q&A]
Hi, 

If I have a "Run" button in a webpage the once the user clicks it would go fetch a table from the backend, how can I have a "spinner" to is shown after pressing the button, and then disappears after the table data is loaded/shown in the frontend UI?

Regards,

**Comment by WolfgangFahl** [comment]
You'll find a progress bar solution that is similar in http://jpdemo.bitplan.com/ bootstrap5 
![grafik](https://user-images.githubusercontent.com/1336221/185737147-f84851d0-f5d4-4bb2-b088-2ed1986af7dc.png)
The source code is in https://github.com/WolfgangFahl/pyJustpyWidgets/blob/main/jpwidgets/bt5widgets.py
